web/wp-content/themes/dctx/index.php

```diff
@@ -1,21 +1,29 @@
-<?php get_header(); ?>
+<?php 
```

Comment:
:no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).